run/test/specs/locators/home.ts

```diff
@@ -1,6 +1,27 @@
 import { StrategyExtractionObj } from '../../../types/testing';
-import { LocatorsInterface } from './index';
+import { LocatorsInterface, LocatorsInterfaceScreenshot } from './index';
+import * as path from 'path';
 
+export class EmptyLandingPage extends LocatorsInterfaceScreenshot {
+  public build() {
+    switch (this.platform) {
+      case 'android':
+        return {
+          strategy: 'id',
+          selector: 'network.loki.messenger:id/emptyStateContainer',
+        } as const;
+      case 'ios':
+        return {
+          strategy: 'accessibility id',
+          selector: 'Empty list',
+        } as const;
+    }
+  }
+  // The landing page has two different states depending on the onboarding flow taken
+  public screenshotFileName(state: 'new_account' | 'restore_account') {
+    return path.join('run', 'screenshots', `${this.platform}`, `landingpage_${state}.png`);
```

Comment:
no need to make a templated string from a string 
```suggestion
    return path.join('run', 'screenshots', this.platform, `landingpage_${state}.png`);
```